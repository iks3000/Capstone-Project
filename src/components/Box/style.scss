@import '../../assets/styles/_variables.scss';
@import '../../assets/styles/_extends.scss';

.info-box {
    margin: 10px 0 40px;
    &__title {
        @extend %title;
    }
}