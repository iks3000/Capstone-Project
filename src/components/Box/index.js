import Info from '../../components/Info';
import './style.scss';

const Box = ({title, content }) => {
    return (
        <div className="info-box" id="boxComponent">
            <h2 className="info-box__title">{title}</h2>
            <Info text={content} />
        </div>
    );
};

export default Box;