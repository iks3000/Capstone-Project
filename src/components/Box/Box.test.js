import React from "react";
import { shallow } from "enzyme";
import Box from "./index";

it("should render Box component", () => {
  expect(shallow(<Box />)).toMatchSnapshot();
});
