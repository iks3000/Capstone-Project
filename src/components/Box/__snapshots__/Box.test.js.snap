// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Box component 1`] = `
<div
  className="info-box"
  id="boxComponent"
>
  <h2
    className="info-box__title"
  />
  <Info />
</div>
`;
